shared/js/background.js

```diff
@@ -69,6 +69,29 @@ function Background() {
   });
 
   chrome.runtime.onInstalled.addListener(function(details) {
+
+      /* Textile TODO:
+
+      We need to run the following at some point here,
+
+      chrome.identity.getAuthToken({'interactive': false}, function (token) {
+          console.log(token)
+      })
+
+      if we don't have user email in the settings, but perhaps we can always do it cause I think it will
+      just return the token without the interactive ui oauth step if the user has already done that (authorized us).
+
+      NOTE: waiting on the private web extension to pass google chrome store so that i can get the right "app key" and
+      add that to the manifest.json file. Without that, the above getAuthToken request fails cause the app ids
+      dont match.
+
+      PENDING EXTENSION: https://chrome.google.com/webstore/detail/ndphnolegaihkhjmfdbafgkgcemhmmgc/publish-delayed
+
+      - not sure how this will work in firefox, maybe we can only do this in chrome
+      - show the post-install page with a way for them to add email if the above fails
+
+      */
```

Comment:
here's an update and a spot to jump in @andrewxhill @asutula, i'm gonna jump on the little web server we need for this